### PR TITLE
10.1 scenarios 

### PR DESCRIPTION
**Pull request by andriizhegurov-okta**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 
+## 1.1.1
```
Comment (laura-rodriguez) [inline CHANGELOG.md:4]: I suggest calling the upcoming version "vNext" until we define what version we're releasing.

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature [hunk 2]
```diff
@@ -0,0 +1,74 @@
+﻿Feature: 10.1: TOTP Support (Google Authenticator) part 2
+
+  Background:
+    Given configured Authenticators are Password (required), and Google Authenticator (required)
+    And a user named "Mary"
+    And Mary does not have an account in the org
+
+  Scenario: 10.1.4: Mary signs up for an account with Password, setups up required Google AuThenticator by scanning a QR Code
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature:8]:
```suggestion
  Scenario: 10.1.4: Mary signs up for an account with Password, setups up required Google Authenticator by scanning a QR Code

```

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    When She selects Google Authenticator from the list
+    And She scans a QR Code
+    And She selects Next
+    Then the screen changes to receive an input for a TOTP code
+    When She inputs the correct code from her Google Authenticator App
+    And She selects Verify
+#
+    Then she sees the list of optional factors
+    When She selects Email from the list
+	Then she sees a page to input a code
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature:32]: Nit: Fix indentation.

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+	When she inputs the correct code from her email
+    Then she sees the list of optional factors
+    When she selects Skip
+#
+    Then she is redirected to the Root View
+    And she sees a table with her profile info
+    And the cell for the value of email is shown and contains her email
+    And the cell for the value of name is shown and contains her first name and last name
+
+  Scenario: 10.1.5: Mary signs up for an account with Password, setups up required Google AuThenticator by entering a shared secret
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature:42]:
```suggestion
  Scenario: 10.1.5: Mary signs up for an account with Password, setups up required Google Authenticator by entering a shared secret

```

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    And she confirms her Password
+    And she submits the set new password form
+    Then she sees a list of required factors to setup
+    When She selects Google Authenticator from the list
+    And She enters the shared Secret Key into the Google Authenticator App
+    And She selects Next on the screen which is showing the QR code
+    Then the screen changes to receive an input for a TOTP code
+    When She inputs the correct code from her Google Authenticator App
+    And She selects Verify
+#
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature:63]: Nit: Remove this line

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    Then she sees the list of optional factors
+    When She selects Email from the list
+	Then she sees a page to input a code
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/10.1.4-5.TotpSupport.feature:66]: Nit: Fix indentation.

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Steps/Pages/GoogleAuthenticator.cs [hunk 7]
```diff
@@ -0,0 +1,76 @@
+﻿using embedded_auth_with_sdk.E2ETests.PageObjectModels;
+using TechTalk.SpecFlow;
+
+namespace embedded_auth_with_sdk.E2ETests.Steps.Pages
+{
+    [Binding]
+    public class GoogleAuthenticator : BaseTestSteps
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Steps/Pages/GoogleAuthenticator.cs:7]: Should we add a suffix to this class to clarify this is the page/steps?

Comment (andriizhegurov-okta) [reply samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Steps/Pages/GoogleAuthenticator.cs:7]: @laura-rodriguez I would remove this suffix for all the other classes in the namespace instead.

File: samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/ManageController.cs [hunk 8]
```diff
@@ -257,24 +278,70 @@ public async Task<ActionResult> SelectPhoneChallengeMethodAsync(SelectAuthentica
...
+            }
+
+            var verifyAuthenticatorOptions = new VerifyAuthenticatorOptions
+            {
+                Code = code,
+            };
+
+            try
+            {
+                var authnResponse = await _idxClient.VerifyAuthenticatorAsync(verifyAuthenticatorOptions, (IIdxContext)Session["idxContext"]);
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/ManageController.cs:316]: Should this method be called Verify or Challenge?

File: src/Okta.Idx.Sdk/AuthenticatorMethodType.cs [hunk 9]
```diff
@@ -28,7 +28,12 @@ public sealed class AuthenticatorMethodType : StringEnum
         /// <summary>
         /// Voice
         /// </summary>
-        public static AuthenticatorMethodType Password = new AuthenticatorMethodType("password");
+        public static AuthenticatorMethodType Password = new AuthenticatorMethodType("password");
+
+        /// <summary>
+        /// Otp
```
Comment (laura-rodriguez) [inline src/Okta.Idx.Sdk/AuthenticatorMethodType.cs:34]:
```suggestion
        /// OTP
```

File: src/Okta.Idx.Sdk/Okta.Idx.Sdk.csproj [hunk 10]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <LangVersion>7.3</LangVersion>
-    <Version>1.0.1</Version>
+    <Version>1.1.1</Version>
```
Comment (laura-rodriguez) [inline src/Okta.Idx.Sdk/Okta.Idx.Sdk.csproj:6]: See my comment above.

**Review by laura-rodriguez — APPROVED** [review]
Minor comments, LGTM.

**Pull request merged**